<launch>
	<!-- Single Robot System -->
	<include file="$(find gspn_framework_package)/ros/launch/single_robot/single_robot_simulation.launch"/>
	<include file="$(find gspn_framework_package)/ros/launch/single_robot/single_robot_navigation.launch"/>

	<!-- GSPN Servers-->
	<node pkg="gspn_framework_package" ns="robot_1" type="server_1_goto.py" name="action_server_1" output="screen"/>
	<node pkg="gspn_framework_package" ns="robot_1" type="server_2.py" name="action_server_2" output="screen"/>

</launch>
